app/views/createAccount/registrationConfirmation.scala.html

```diff
@@ -31,11 +31,11 @@
 
 @mainLayout(pageTitle = messages("account.confirmation.title"), showTopNavigation = false) {
 
-
-
-        <div class="govuk-body">
-            <h1 class="govuk-heading-xl">@messages("account.confirmation.heading")</h1>
-            <h2 class="govuk-heading-l">@messages("create.success.personId") @id</h2>
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1>@messages("account.confirmation.heading")</h1>
```

Comment:
does the <h1> require a class?

Reply:
@darrell-plant no as it uses outer class styles

Reply:
safer to make it explicit - that is what plat-ui has advised